compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/types/BLangErrorType.java

```diff
@@ -43,9 +42,6 @@ public void accept(BLangNodeVisitor visitor) {
     @Override
     public String toString() {
         StringBuilder val = new StringBuilder(this.type.toString());
-        if (this.inferErrorType) {
-            val.append("<*>");
-        }
         return val.toString();
```

Comment:
We can just do `this.type.toString()` right?

Reply:
yes. Thanks